azimuth/config.py

```diff
@@ -289,6 +335,12 @@ def check_pipeline_names(cls, pipeline_definitions):
         return pipeline_definitions
 
 
+class LanguageConfig(CommonFieldsConfig):
+    # Language configuration sets multiple config values; see reference dictionary for details
```

Comment:
Now that we moved the `root_validator` away, maybe it's not so clear what the "reference dictionary" is?
```suggestion
    # Language configuration sets multiple config values; see `config_defaults_per_language` for details
```